src/webloader.js

```diff
@@ -84,6 +84,28 @@ function handlePushState(client) {
 	window.addEventListener('stateupdate_evolv', updateContext);
 }
 
+function checkInstanceCount(evolv) {
+	if (!evolv.instancesCount) {
+		evolv.instancesCount = 1;
+	} else {
+		evolv.instancesCount++;
+	}
+
+	// Not works for IE, but it needs only in web-editor (chrome)
+	if (window.CustomEvent) {
+		try {
+			const webloaderLoadEvent = new CustomEvent('webloader-loaded', {'detail': 	evolv.instancesCount });
+			document.dispatchEvent(webloaderLoadEvent);
+		} catch(e) {
+			console.warn('Evolv: Could not fire custom event')
+		}
+	}
+
+	if (evolv.instancesCount > 1) {
+		throw new Error('Multiple Evolv instances - please verify you have only loaded Evolv once');
```

Comment:
I might possibly advise against throwing a bona fide error. One customer (Blinds, if I recall) had some mechanism that relayed errors to their monitoring system. They were unhappy when we throw errors that weren't fatal to their whole page.

Reply:
https://evolvai.atlassian.net/browse/RKT-5989